src/umpire/ResourceManager.cpp

```diff
@@ -569,4 +570,15 @@ ResourceManager::getNextId() noexcept
   return m_id++;
 }
 
+void print_allocator_records(Allocator alloc, std::ostream& os) {
```

Comment:
Can you move this to Umpire.hpp/cpp, rather than the ResourceManager implementation file. It's not a class method, so shouldn't be grouped in here.

Reply:
Done!